tests/topotests/pytest.ini

```diff
@@ -18,7 +18,7 @@ norecursedirs = .git example-test example-topojson-test lib docker
 # show_router_config = True
 
 # Default daemons binaries path.
-#frrdir = /usr/lib/frr
+frrdir = /usr/local/sbin
 #quaggadir = /usr/lib/quagga
```

Comment:
Are you sure of that ? Defaut binary has moved ? I think it is preferable to keep frrdir commented or if not to setup it to the default /usr/lib/frr

Reply:
Got confirmation from CI/CD that you must revert this modification

Reply:
Fixed! As you can imagine I committed that change by accident.